src/qml/main.qml

```diff
@@ -106,6 +107,12 @@ Application {
                     height: DeviceInfo.hasSpeaker ? Dims.h(16) : 0
                     clip: true
                 }
+                ListItem {
```

Comment:
Don't you have weird scrolling issues if you don't update "var nb = 13" to 14 on line 58 ?

`elementsNb` is really awful, we should really rework this someday but in the meantime I think we have to keep this counter accurate... :(

Reply:
That I can fix.  See https://github.com/AsteroidOS/asteroid-settings/issues/53